doc/design.rst

```diff
@@ -606,3 +597,61 @@ are deleted, the particular snapshot vanished and all snapshots
 depending on data that has been added in the snapshot cannot be restored
 completely. Restic is not designed to detect this attack.
 
+With the aforementioned assumptions and guarantees in mind, the following  are
+examples of things an adversary could achieve in various circumstances.
+
+An adversary with access to your backup storage location could: 
+- Delete or manipulate your backups, thereby impairing your ability to restore
+files from the compromised storage location. - Attempt a brute force password 
+guessing attack against a copy of the  repository (even more reason to use 
+long, random passwords).
+
+An adversary with write access to your files at the storage location could: 
+- Delete specific files (by using timestmaps to determine which files belong to
+what snapshots), resulting in snapshots vanishing and all dependent snapshots
+from being completely restored.
+
+An adversary who can passively monitor all Internet traffic could: 
+- Determine from where you create your backups (i.e., the location where the 
+requests originate).
+- Determine where you are storing your remote backups via HTTP headers (URLs)
```

Comment:
Does "where" just refer to the path on the target system? No matter whether you use encryption an adversary will be able to tell which server you contacted.

Reply:
Good question. In my mind, the "where" includes any details leaked in the URL. At minimum, this includes the server you contacted. I agree an adversary can tell which server you contacted regardless if encryption is in play or not. While that should generally be understood, I thought it was worth explicitly calling out as something to consider. If someone is very concerned about this, backing up on a local network rather than over the Internet should be considered. What change/suggestion did you have in mind?